content/blog/pycon-india-tickets-2018.md

```diff
@@ -0,0 +1,34 @@
+Title: PyCon India Tickets 2018
+Date: 2018-05-31 12:45
+Category: 2018
+Slug: pycon-india-tickets-2018
+Summary: Details about the different tickets for PyCon India 2018.
+
+PyCon India 2018, the tenth edition of the premier conference in India on using and developing the Python programming language is conducted annually by the Python developer community has started ticket sales for the mega event that is scheduled from 5th - 9th October 2018. 
+
+The conference will be having a tight schedule of workshops,  keynotes, talks, workshops, and dev-sprints strung together through 3 different tracks. Any person who has a conference ticket is eligible to attend the Devsprints. Although the Developer Sprints are free, we would be having registrations beforehand to manage the logistics. The ticket structure is as follows:- 
+
+**Grab the early bird tickets now from [Explara](https://in.explara.com/e/pycon-india-2018/).**
+
+##Tickets Structure##
+
+### Early Bird Tickets ###
+An Early bird ticket reserves your seat for the conference on 6th - 7th October, 2018, with a pricing of INR 1200 (GST applicable), the sale of early bird tickets ends on 30 June 2018 or completion of 300 tickets. 
+
+**UPDATE: 200 early-bird tickets already sold**
+
+### Student Ticket ###
+A student ticket reserves your seat for the conference on 6th - 7th October, 2018, with a pricing of INR 1000 (GST applicable), the sale of student tickets ends on 30 September 2018 or completion of 300 tickets. 
+
+### Regular Ticket  
+A regular ticket reserves your seat for the conference on 6th - 7th October, 2018, with a pricing of INR 1700 (GST applicable), the sale of regular tickets begins on 1st July 2018.
+
+### Late Bird Ticket ###
+A late bird ticket reserves your seat for the conference on 6th - 7th October, 2018, with a pricing of INR 2500 (GST applicable), the sale of late bird begins on 1st September 2018.
+
+###Payment Options###
+You can pay through Citrus or EBS gateway via Credit Card, Debit Card or Netbanking. You can also use Paytm for your registration by accepting the terms and conditions of Explara.
+
+**See you there at the Conference!!**
+
+_Want to come for the conference, but not sure how to pitch the same to your boss. Leave it to us, check out [Convince My Boss](http://www.in.pycon.org/convince-my-boss.html)._
```

Comment:
@vipulgupta2048 Th correct link of the website is https://in.pycon.org/2018/ . Please fix this